src/auth/auth.hpp

```diff
@@ -213,6 +214,13 @@ class Auth final {
    */
   bool AccessControlled() const;
 
+  /**
+   * Returns whether the access is controlled by authentication/authorization carried out by an external module.
+   *
+   * @return `true` if auth needs to run
+   */
+  bool UsingAuthModule() const { return !modules_.empty(); }
```

Comment:
Thanks for the detailed explanation! I’ve applied your suggestions so that users logged in via module can modify Memgraph-internal user data, e.g. run `CREATE USER` or `SHOW USERS`.